### PR TITLE
docs: date format to YYYY-MM-DD

### PR DESCRIPTION
**Pull request by atrisovic**
Signed-off-by: Ana Trisovic <ana.trisovic@gmail.com>

### CODE REVIEW

File: cernopendata/modules/fixtures/data/docs/cod-welcome/cod-welcome.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
             {"primary": "News" }
         ],
         "author": "Open Data Portal team",
-        "created": "November 15th 2017",
+        "created": "2017-11-17",
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/cod-welcome/cod-welcome.json]: Guessing this was changed intentionally from 15 November to 17 November, correct?

Comment (atrisovic) [reply cernopendata/modules/fixtures/data/docs/cod-welcome/cod-welcome.json]: yup :) closer to the release

Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/docs/cod-welcome/cod-welcome.json]: ... that would have been `2017-12-17` 😄

Comment (atrisovic) [reply cernopendata/modules/fixtures/data/docs/cod-welcome/cod-welcome.json]: omg yes 😂

**Comment by tiborsimko** [comment]
LGTM. We may want to check and update the displayer, so that "2017-12-16" would be displayed as "16 December 2017".

**Pull request merged**